user/views.py

```diff
@@ -1,3 +1,127 @@
-from django.shortcuts import render
+from django.http import JsonResponse, HttpResponse
+from django.core.validators import RegexValidator
+from django.core.exceptions import ValidationError
+from partner.models import Counselor, Product
+from .models import History, User
+from trost.settings import SECRET_KEY
+from django.views import View
+from .models import User
+import bcrypt
+import names
+import json
+import jwt
```

Comment:
임포트 순서는 파이썬 라이브러리를 제일위에

장고모듈을 그다음에

사용자화된 클래스나 함수를 제일 나중에 간격 조정해서 배치해주세요.
아래 예시대로 해주시고 
종류별로 한줄씩 간격줘서 구분지어주세요.
```suggestion
import jwt

from django.http import JsonResponse, HttpResponse

from .models import History, User

```